### PR TITLE
Card Secondary Variant

### LINKED ISSUE #2658: Card secondary variant

### PR DESCRIPTION
**Pull request by lilyvc**
closes #2658

### CODE REVIEW

**Review by Fercas123 — CHANGES_REQUESTED** [review]
looks good! just 1 repeated class

File: packages/core/src/card/InteractableCard.css [hunk 1]
```diff
@@ -11,6 +11,9 @@
   padding: var(--saltCard-padding, var(--salt-spacing-300));
 }
 
+.saltInteractableCard-secondary {
+  background: var(--salt-container-secondary-background);
+}
```
[the remark this answers is not in the document]
Comment (lilyvc) [reply packages/core/src/card/InteractableCard.css:16]: thanks for noticing!

**Review by Fercas123 — DISMISSED** [review]

**Review by ImmanuelBaskaran — APPROVED** [review]

**Review by feliciackh — DISMISSED** [review]

File: packages/core/src/card/Card.css [hunk 2]
```diff
@@ -8,6 +8,11 @@
   padding: var(--saltCard-padding, var(--salt-spacing-300));
 }
 
+.saltCard-secondary {
```
Comment (origami-z) [inline packages/core/src/card/Card.css:11]: Should `primary` variant sharing same API?

Comment (lilyvc) [reply packages/core/src/card/Card.css:11]: yeah good point, I will update this

**Comment by alycrys** [comment]
Would it make sense to add an example to storybook also? Maybe to QA story for chromatic testing? Not sure what will happen to the storybook docs, so maybe we don't need an example there.

File: packages/core/src/card/Card.css (hunk 2 above)
Comment (origami-z) [reply packages/core/src/card/Card.css:14]: `--saltCard-background` not the same after update?

File: packages/core/src/card/InteractableCard.css [hunk 3]
```diff
@@ -1,16 +1,25 @@
 /* Styles applied to InteractableCard */
 .saltInteractableCard {
-  background: var(--saltCard-background, var(--salt-container-primary-background));
```
Comment (origami-z) [inline packages/core/src/card/InteractableCard.css:3]: `--saltCard-background` will be gone?

File: packages/core/src/card/InteractableCard.css [hunk 4]
```diff
@@ -64,14 +73,30 @@ a:link .saltInteractableCard * {
...
+  border-color: var(--saltCard-borderColor-disabled, var(--salt-container-primary-borderColor-disabled));
+  background: var(--saltCard-background-disabled, var(--salt-container-primary-background-disabled));
+}
+
+/* Styles applied to InteractableCard secondary variant if `disabled={true}` */
+.saltInteractableCard-secondary.saltInteractableCard-disabled,
+.saltInteractableCard-secondary.saltInteractableCard-disabled:focus,
+.saltInteractableCard-secondary.saltInteractableCard-disabled:hover,
+.saltInteractableCard-secondary.saltInteractableCard-disabled:active {
+  border-color: var(--salt-container-secondary-borderColor-disabled);
```
Comment (origami-z) [inline packages/core/src/card/InteractableCard.css:96]: Different API here as well

File: packages/core/src/card/Card.css (hunk 2 above)
Comment (lilyvc) [reply packages/core/src/card/Card.css:14]: I've updated - is this what you mean? or should it be a different token like:

```
background: var(--saltCard-secondary-background, var(--salt-container-secondary-background));
```

**Review by Fercas123 — APPROVED** [review]

**Review by feliciackh — APPROVED** [review]

**Pull request merged**